src/database.rs

```diff
@@ -112,17 +112,28 @@ impl Database {
     }
 
     pub async fn insert_image(&self, image: &Image) -> DBResult<()> {
-        sqlx::query("INSERT INTO images (id, user_address, url, thumbnail_url, metadata) VALUES ($1, $2, $3, $4, $5)")
+        sqlx::query("INSERT INTO images (id, user_address, url, thumbnail_url, is_public, metadata) VALUES ($1, $2, $3, $4, $5, $6)")
             .bind(parse_uuid(&image.id)?)
             .bind(&image.metadata.user_address.to_lowercase())
```

Comment:
I think it is suggesting this without the `$` because does not needed to be borrowed, can we check if the compiler complains after removing that? same feedback should be applied below
```suggestion
            .bind(image.metadata.user_address.to_lowercase())
```